autorest/multiapi/templates/multiapi_profiles_vendor.py.jinja2

```diff
@@ -0,0 +1,367 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm using what's in `azure-common` master, not @xiangyan99's PR. Wanted to change as little as possible

Reply:
I put all of the [`profiles`](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/core/azure-common/azure/profiles) files into one